dockers/docker-fpm-quagga/start.sh

```diff
@@ -18,6 +18,7 @@ echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
 rm -f /var/run/rsyslogd.pid
 
+supervisorctl start bgp_eoiu_marker
```

Comment:
This needs to be configurable and not start by default.

Reply:
thanks, I plan to drop the support for quagga, and enable it for frr only.

Reply:
Even for frr, this need to be configurable. We can't have this starting by default and doing cli scraping every second.

Reply:
Not sure if you checked the script.  
It exit immediately if it is cold start.  For warm start, it exits itself after collecting eor for all neighbors or after 120 seconds.

Reply:
Even for warm start this should be configurable.